projects/storefrontlib/cms-structure/seo/structured-data/json-ld.directive.ts

```diff
@@ -27,25 +26,27 @@ export class JsonLdDirective {
    * Writes the schema data to a json-ld script element.
    */
   @Input() set cxJsonLd(schema: string | {}) {
-    this.jsonLD = this.generateJsonLdScript(schema);
+    this.generateJsonLdScript(schema);
   }
 
-  @HostBinding('innerHTML') jsonLD: SafeHtml | undefined;
-
   constructor(
+    private renderer: Renderer2,
     protected jsonLdScriptFactory: JsonLdScriptFactory,
-    protected sanitizer: DomSanitizer
+    private element: ElementRef
```

Comment:
Since we are using `protected` everywhere (for extensibility reasons), I'm curious is there a reason why we are switching to `private`?

Reply:
yes it should be 'protected'. just thought originally it was safer.